neptune/management/internal/api.py

```diff
@@ -141,37 +143,43 @@ def create_project(
     description: Optional[str] = None,
     api_token: Optional[str] = None,
 ) -> str:
-    """Creates a new project in your Neptune workspace.
+    """Creates a new project in a Neptune workspace.
+
     Args:
-        name(str): The name of the project in Neptune in the format 'WORKSPACE/PROJECT'.
-            If workspace argument was set, it should only contain 'PROJECT' instead of 'WORKSPACE/PROJECT'.
-        key(str): Project identifier. It has to be contain 1-10 upper case letters or numbers.
-            For example, 'GOOD5'
-        workspace(str, optional): Name of your Neptune workspace.
-            If you specify it, change the format of the name argument to 'PROJECT' instead of 'WORKSPACE/PROJECT'.
-            If 'None' it will be parsed from the `name` argument.
-        visibility(str, optional): level of visibility you want your project to have.
+        name: The name of the project in Neptune in the form 'workspace-name/project-name'.
+            If workspace argument was set, it should only contain 'project' instead of 'workspace-name/project-name'.
```

Comment:
Should be 'project-name' instead of 'project', right?

Reply:
Yep, think it's a leftover from some find/replace change. Thanks for spotting